doc/api/process.md

```diff
@@ -1651,7 +1651,9 @@ added: v0.2.0
 * {Object}
 
 The `process.versions` property returns an object listing the version strings of
-Node.js and its dependencies.
+Node.js and its dependencies. In addition, `process.versions.modules` indicates
+the current ABI version, which is increased whenever a C++ API changes. Node.js
+will refuse to load native modules built for an older `modules` value.
```

Comment:
https://github.com/nodejs/node/blob/e03ee719e6e760777b237861bf0b7835432af8bb/src/node_version.h#L46-L52

Documentation text mirrors the only available documentation for this property.... an in-source comment. @addaleax are you saying the comment is wrong?

If so, this PR should change both the comment and the docs.